etna/whatson/models.py

```diff
@@ -685,6 +685,34 @@ def primary_access_type(self):
         if primary_access := self.event_access_types.first():
             return primary_access.access_type
 
+    @cached_property
+    def date_time_range(self):
+        delta = self.end_date - self.start_date
```

Comment:
Suggestion: move this to after the first if clause so we only calculate it if we need it

Reply:
This is removed now. I have added the format variables here instead - I think it reads clearer to declare them all together at the start rather than as they are needed, but let me know if you'd like me to change that.